docs/notebooks/quickstart.ipynb

```diff
@@ -61,7 +61,9 @@
         "id": "Xpy1dSgNqCP4"
       },
       "source": [
-        "We'll be generating random data in the following examples. One big difference between NumPy and JAX is how you generate random numbers. For more details, see [Common Gotchas in JAX](Common_Gotchas_in_JAX.ipynb#🔪-Random-Numbers)."
+        "We'll be generating random data in the following examples. One big difference between NumPy and JAX is how you generate random numbers. For more details, see [Common Gotchas in JAX].\n",
+        "\n",
+        "[Common Gotchas in JAX]: https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#%F0%9F%94%AA-Random-Numbers"
```

Comment:
I think I have misunderstood your comment. I still see the link in the CI-generated docs here: https://jax--4102.org.readthedocs.build/en/4102/notebooks/quickstart.html#Multiplying-Matrices

I switched to reference style links just because the line was getting a little unwieldily but happy to revert to the inline version.

Reply:
Ah, my mistake, I misread the diff. Thanks!